test/e2e-test/testdata/app/app5.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  annotations:
+    "app.oam.dev/rollout-template": "false"
```

Comment:
What does "false" mean? I think it's better to add a webhook to check instead of allowing this.

Reply:
Because some users assume that the false  annotations means inplace upgrade